algebra/src/curves/models/mod.rs

```diff
@@ -55,7 +55,11 @@ pub trait SWModelParameters: ModelParameters {
 
         let mut result = 4;
         for r in &RECOMMENDATIONS {
-            if num_scalars > *r { result += 1 } else { break }
+            if num_scalars > *r {
+                result += 1
+            } else {
+                break;
+            }
```

Comment:
It's be cool if some rustfmt option prevented this, but whatever

Reply:
Yeah I was searching for one, but couldn't find it =/